docs/assets/includes/ai-info.md

```diff
@@ -0,0 +1,5 @@
+!!! note
+    -   This feature is compatible with most programming languages and requires no additional setup.
+    -   Suggestions are generated using the description of the static analysis issue and a few lines of surrounding code.
+    -   This feature leverages the OpenAI API to generate suggestions. Please refer to the [OpenAi API data usage policies](https://openai.com/policies/api-data-usage-policies) for more information.
+    -   No information is shared with other third parties or used to train AI models.
```

Comment:
I think it makes sense to merge these two items since they're closely related:

```suggestion
    -   This feature leverages the OpenAI API to generate suggestions, but no information is shared with other third parties or used to train AI models. Please refer to the [OpenAI API data usage policies](https://openai.com/policies/api-data-usage-policies) for more information.

```

(My suggestion also adds the missing blank line. :wink:)